phpstan.neon.dist

```diff
@@ -0,0 +1,16 @@
+parameters:
+    parallel:
+        processTimeout: 300.0
+    level: 1
+    paths:
+        - ./
+    excludes_analyse:
+        - vendor/
+        - fonts/
+
+    # TODO review once we drop PHP 7.x support
+    treatPhpDocTypesAsCertain: false
```

Comment:
This is not needed, I did remove it on the fork as it made not difference

Reply:
config simplified